src/Language/PureScript/Docs/Prim.hs

```diff
@@ -258,12 +258,22 @@ kindSymbol = primType "Symbol" $ T.unlines
   , ""
   , "Construct types of this kind using the same literal syntax as documented"
   , "for strings."
+  , ""
+  , "    type Hello :: Symbol {- String -}"
```

Comment:
Did you mean to leave the `{- String -}` comment in here? I'm not sure that's helpful; it almost reads like you're saying the two are interchangeable.